fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/MigrateV0021AbstractTypeRemoval.java

```diff
@@ -82,21 +77,9 @@ public MigrateV0021AbstractTypeRemoval(IDatabaseAdapter adapter, String adminSch
 
     @Override
     public void run(IDatabaseTranslator translator, Connection c) {
```

Comment:
Not for this PR, but I wonder if our 5.0 is a chance to remove some of these historical migration objects.
Or, if not, maybe we deprecate them and remove in 6.0 (so we could support 4.x -> 5.x but we'd require 5.x to get to 6.x)